app/src/main/res/drawable/ic_watch.xml

```diff
@@ -3,7 +3,12 @@
     android:height="24dp"
     android:viewportWidth="24"
     android:viewportHeight="24">
+  <group
+    android:pivotX="12"
+    android:pivotY="12"
+    >
   <path
       android:pathData="M9,16.2L4.8,12l-1.4,1.4L9,19 21,7l-1.4,-1.4L9,16.2z"
       android:fillColor="#fff"/>
+  </group>
```

Comment:
I'm not sure why the group was required but it's missing in the ic_watch_red.xml file and the XML file should be formatted.

Reply:
I think this may have been leftovers from something that failed. I'll revert to the master copy and see what happens